Dashboard/app/js/templates/navSurveys/question-view.handlebars

```diff
@@ -62,9 +62,8 @@
   {{/if}}
 
     <!-- Question specific material -->
-<div class="questionOption floats-in">
-
     {{#if view.amOptionType}}
+      <div class="questionOption floats-in">
```

Comment:
Does this need to be inside each individual `{{if}}` block?  Why not outside the block like it was so that its not repeated over and over?